package.json

```diff
@@ -5,7 +5,6 @@
 	"dependencies": {},
 	"devDependencies": {
 		"@fortawesome/fontawesome-free": "^5.13.0",
-		"TYPO3CMS-Guide-HowToDocument": "github:TYPO3-Documentation/TYPO3CMS-Guide-HowToDocument",
```

Comment:
Not sure what this was being used for. Removed it because it causes `npm install` to fail.